src/app/assets/languages/en.json

```diff
@@ -198,7 +198,7 @@
           "transactional-disabled": "This cache can not be transactional",
           "transactional-disabled-description": "Transactional caches are available in synchronous replication mode only. Remove the capability or chose synchronous replication.",
           "backups-disabled": "Backups are disabled",
-          "backups-disabled-description": "Backups are available only when you configure {{brandname}} to perform cross-site replication. Remove the capability or enable cross-site replication.",
+          "backups-disabled-description": "Backups are available only when you configure {{brandname}} to perform cross-site replication. To add a backup, enable cross-site replication. Otherwise, remove the capability.",
```

Comment:
It's a complete backup configuration. Users add backup sites, choose backup strategy, merge policy, timeouts, and much more.

Reply:
Ah, I see. Rolling this back in my next commit, then!